x/profiles/keeper/keeper_chain_links.go

```diff
@@ -95,3 +94,64 @@ func (k Keeper) DeleteAllUserChainLinks(ctx sdk.Context, user string) {
 		k.DeleteChainLink(ctx, link)
 	}
 }
+
+// getOldestUserChainByChain returns the oldest chain link of the given owner associated to the given chain name
+// If the such the chain link is not exists, return false instead.
```

Comment:
```suggestion
// getOldestUserChainByChain returns the oldest chain link of the given owner associated to the given chain name.
// If such chain link does not exist, returns false instead.
```

I would also rename this to `getOldestUserChainLink`